sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -69,16 +68,23 @@ def __call__(self, y_true, y_pred):
         Returns
         -------
         loss : float
-            Calculated loss metric
+            Calculated loss metric. If `greater_is_better` attribute is True
+            returns the negative of the metric. If `greater_is_better` attribute
+            is False the metric is returned.
         """
-        return self._func(y_true, y_pred)
+        sign = 1 if self.greater_is_better else -1
+        return sign * self._func(y_true, y_pred)
```

Comment:
Can we undo this change? 

We currently use `greater_is_better` directly when selecting results in tuning. 

https://github.com/alan-turing-institute/sktime/blob/7343dc36566935b42167878c45d570099af96dab/sktime/forecasting/model_selection/_tune.py#L287-L288

That seems to be the only place where we currently use `greater_is_better`.